### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by rprouse**
This is the recommended text from the CC website. Do we need more?

### CODE REVIEW

**Review by jnm2 — APPROVED** [review]
I'm good with BY-SA.

**Comment by CharliePoole** [comment]
I like the license choice, but the text seems to be for a footer, as we see on many websites. Since these are not web pages, should we put it at the end of each doc? Do they give any guidance on how to apply it?

**Comment by rprouse** [comment]
I followed the guidance at https://creativecommons.org/choose/results-one?license_code=by-sa&amp;jurisdiction=&amp;version=4.0&amp;lang=en but it isn't very specific on how to apply it to a repository, so I went with the GitHub way of a `LICENSE.md` file.

At this point adding it to each doc won't be hard, but I don't think we want to do that to every file in the other repositories that we are considering the CC BY-SA license for. Based on that I thought this was a good compromise. We could also add it to the base of the README to make it more visible.

**Comment by CharliePoole** [comment]
If it seems OK based on what they say, then it's OK with me.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/4.0/ or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
```
Comment (CharliePoole) [inline LICENSE.md:1]: How about "The documents in this repository are licensed..."

**Review by CharliePoole — APPROVED** [review]
I like my rewording better, but I'm not fussy.

**Review by ChrisMaddock — APPROVED** [review]

**Pull request merged**